data/human/human missions.txt

```diff
@@ -6891,3 +6891,370 @@ mission "Returning to Paradise"
+	name "Take the film to <planet>"
+	description "Take the film to <destination>, where the rest of the Adelita group remained behind. (You will be paid <payment> after you complete the entire job.)"
+	landing
+	passengers 1
+	cargo "stealth drones" 1
+	infiltrating
+	"apparent payment" 200000
+	source "Hippocrates"
+	destination "Maker"
+	to offer
+		has "Adelita 3: done"
+
+	on offer
+		conversation
+			branch "player has a fleet"
+				"total ships" - ( "ships: Fighter" + "ships: Drone" ) > 1
+
+			`It's not hard to find a remote canyon within range of the target area. You deactivate your shields but keep the <ship> hovering in case you need to make a quick escape. You are low enough that Bunker can release the drones from the open cargo bay without a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
+				goto "on site"
+
+			label "player has a fleet"
+			`You carefully bring the <ship> into the atmosphere while the rest of your fleet stays in space so that the pirates won't catch wind of Adelita's operation. Luckily, it's not hard to find a remote canyon within range of the target area for you to lie low. You deactivate your shields but keep the <ship> hovering in case you need to make a quick escape while Bunker releases the drones from the open cargo bay. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a display and sits at the controller. "Okay, they're on autopilot to the rendezvous site."`
+
+			label "on site"
+			`	As you watch the display, the drones arrive on site, and you can see a lone Syndicate-built freighter with a security detail posted in front of the loading ramp. "The pirates must be late," says Bunker, smirking. "Typical. I'm going to scatter the drones and set a proximity alert."`
+			`	He hums to himself for a while before he turns to you and starts talking again. "You know, there's a part of me that would much rather be liberating slaves instead of filming them, but for now it seems like this is the best way for me to help. Say we did liberate them right now; there's just going to be another slave ship tomorrow. But if we create political and social pressure, that could stop the slave ships forever."`
+			choice
+				`	"I think you're right."`
+				`	"What about doing both?"`
+					goto plan
+
+			`	"And you know they'll use those poor people as human shields. Say we-" A chirp from the proximity alert cuts him off.`
+				goto showtime
+
+			label plan
+			`	"Now you're talking! But it would take a lot of planning to do it without killing the people we are trying to liberate. You know they'll use those poor people as human shields. Say we-" A chirp from the proximity alert cuts him off.`
```

Comment:
```suggestion
			`	"Now you're talking! But it would take a lot of planning to do it without killing the people we're trying to liberate. You know they'll use those poor people as human shields. Say we-" A chirp from the proximity alert cuts him off.`
```